generators/fix-entity/index.js

```diff
@@ -153,16 +153,14 @@ module.exports = generator.extend({
             jhipsterFunc.replaceContent(paramFiles.liquibaseEntity, `<createTable tableName="${this.entityTableName}`, `<createTable tableName="${newValue}`);
         };
 
-        this.log(chalk.bold.yellow('writing'));
-
         // verify files exist
-        for (const file in files) {
-            // hasOwnProperty to avoid inherited properties
-            if (files.hasOwnProperty(file) && !fs.existsSync(files[file])) {
+        filesArr.forEach((file) => {
+            if (!fs.existsSync(files[file])) {
                 throw new Error(`JHipster-db-helper : File not found (${file}: ${files[file]}).`);
             }
-        }
+        });
 
+        // Add/Change/Keep dbhColumnName for each field
```

Comment:
I think you really enjoy commenting lines. But I really enjoy fixing my mistakes though.